src/SearchResults.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import fakeBookings from "./data/fakeBookings.json";
+import moment from "moment";
+
+const SearchResults = (props) => (
+  <table className="table table-striped">
+    <CreateTableHeader />
+    <CreateEachRow results={fakeBookings} />
+  </table>
+);
+
+const CreateTableHeader = () => (
```

Comment:
Loving the small components!! So easy to see whats going on 😄 

I would just suggest that you don't need to add the prefix `Create`. 
You're right, this does create a table header, but its not the convention to add it